src/Chapter1.hs

```diff
@@ -620,9 +632,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let units = mod (abs n) 10
+      tens = mod (div (abs n) 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`s, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use when you do `mod (abs n) 10` and `div (abs n) 10`.

So you could write it this way:

```haskell
(d, m) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂